parm/config/config.ocnanalrun

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -x
```

Comment:
Other config files aren't turning off trace on their own, so this one shouldn't either.